### PR TITLE
adding description for support of product arrays with Snap CAPI

### PR DESCRIPTION
**Pull request by joe-ayoub-segment**
### Proposed changes

Snap CAPI Destination needs to allow customers to pass arrays of products in conversion events. 
This capability is being added with this [action-destinations PR](https://github.com/segmentio/action-destinations/pull/1557).  

Once deployed, this docs update explains the change. 

### Merge timing
- Merge only after the PR above has been deployed. 

### Related issues (optional)
https://segment.atlassian.net/browse/PE-88

### CODE REVIEW

File: src/connections/destinations/catalog/actions-snap-conversions/index.md [hunk 1]
```diff
@@ -70,6 +70,11 @@ If you want to send a Snap Event Type that Segment doesn’t have a prebuilt map
 
 The Snapchat Conversions API only supports sending Event Types that are in the [predefined `event_type` list](https://marketingapi.snapchat.com/docs/conversion.html#conversion-parameters){:target="_blank"}. This includes custom events. You must use `CUSTOM_EVENT_1`, `CUSTOM_EVENT_2`, `CUSTOM_EVENT_3`, `CUSTOM_EVENT_4`, or `CUSTOM_EVENT_5` as the Event Type. Events sent with an invalid event type will fail with an `Unrecognized event type` error.
 
+### Single or multiple products or items
+It's possible to send details of either single or multiple products / items in a single conversion event.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-snap-conversions/index.md:74]:
```suggestion
It's possible to send details of either single or multiple products/items in a single conversion event. 
```

File: src/connections/destinations/catalog/actions-snap-conversions/index.md [hunk 2]
```diff
@@ -70,6 +70,11 @@ If you want to send a Snap Event Type that Segment doesn’t have a prebuilt map
 
 The Snapchat Conversions API only supports sending Event Types that are in the [predefined `event_type` list](https://marketingapi.snapchat.com/docs/conversion.html#conversion-parameters){:target="_blank"}. This includes custom events. You must use `CUSTOM_EVENT_1`, `CUSTOM_EVENT_2`, `CUSTOM_EVENT_3`, `CUSTOM_EVENT_4`, or `CUSTOM_EVENT_5` as the Event Type. Events sent with an invalid event type will fail with an `Unrecognized event type` error.
 
+### Single or multiple products or items
+It's possible to send details of either single or multiple products / items in a single conversion event. 
+Single product / item: Use the 'Item ID', 'Item Category', 'Brand', 'Number of Items' and 'Price' fields.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-snap-conversions/index.md:75]:
```suggestion
- **Single product/item**: Use the "Item ID", "Item Category", 'Brand', "Number of Items" and "Price" fields. 
```

File: src/connections/destinations/catalog/actions-snap-conversions/index.md [hunk 3]
```diff
@@ -70,6 +70,11 @@ If you want to send a Snap Event Type that Segment doesn’t have a prebuilt map
 
 The Snapchat Conversions API only supports sending Event Types that are in the [predefined `event_type` list](https://marketingapi.snapchat.com/docs/conversion.html#conversion-parameters){:target="_blank"}. This includes custom events. You must use `CUSTOM_EVENT_1`, `CUSTOM_EVENT_2`, `CUSTOM_EVENT_3`, `CUSTOM_EVENT_4`, or `CUSTOM_EVENT_5` as the Event Type. Events sent with an invalid event type will fail with an `Unrecognized event type` error.
 
+### Single or multiple products or items
+It's possible to send details of either single or multiple products / items in a single conversion event. 
+Single product / item: Use the 'Item ID', 'Item Category', 'Brand', 'Number of Items' and 'Price' fields. 
+Multiple products / items: Use the 'Products' field which accepts an array of products / items.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-snap-conversions/index.md:76]:
```suggestion
- **Multiple products/items**: Use the "Products" field which accepts an array of products / items.  
```

**Review by forstisabella — APPROVED** [review]

**Comment by forstisabella** [comment]
@joe-ayoub-segment - Is this PR ready to merge?

**Comment by joe-ayoub-segment** [comment]
Hi @forstisabella , we just deployed the Snap change a moment ago, so yes you can deploy this doc change please!

**Pull request merged**